packages/frontend/core/src/components/affine/setting-modal/setting-sidebar/style.css.ts

```diff
@@ -129,13 +128,20 @@ globalStyle(`${accountButton} .content`, {
   flexGrow: '1',
   minWidth: 0,
 });
+globalStyle(`${accountButton} .name-container`, {
+  display: 'flex',
+  justifyContent: 'flex-start',
+  alignItems: 'center',
+  width: '100%',
+  gap: '4px',
+});
 globalStyle(`${accountButton} .name`, {
   fontSize: 'var(--affine-font-sm)',
   fontWeight: 600,
   overflow: 'hidden',
   textOverflow: 'ellipsis',
   whiteSpace: 'nowrap',
-  flexGrow: 1,
+  // flexGrow: 1,
```

Comment:
The height is not correct, should be 22px
![image](https://github.com/toeverything/AFFiNE/assets/9527419/7cca4db4-170b-4f20-afb1-c50899f62f37)